src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,135 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+import duke.ui.Ui;
+
+
+/**
+ * Task List to store all the tasks
+ */
+public class TaskList {
+
+    protected ArrayList<Task> taskList;
+
+    public TaskList() {
+        this.taskList = new ArrayList<>();
+    }
+
+    /**
+     * Initialises the task list
+     *
+     * @param file The file storing the task information
+     */
+    public TaskList(File file) {
+        this.taskList = new ArrayList<>();
+        try {
+            Scanner s = new Scanner(file); // create a Scanner using the File as the source
+            while (s.hasNext()) {
+                String cur = s.nextLine();
```

Comment:
Perhaps could have avoided the short form and have used a slightly more descriptive variable name (e.g. `currentLine`)?

I also noticed similar instances in several other places as well...